client/src/test/java/com/sinch/sdk/e2e/domains/WebhooksHelper.java

```diff
@@ -0,0 +1,58 @@
+package com.sinch.sdk.e2e.domains;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+
+public class WebhooksHelper {
+
+  public static <T> Response<T> callURL(URL url, Function<String, T> parseEvent)
+      throws IOException {
+
+    HttpURLConnection con = (HttpURLConnection) url.openConnection();
+    con.setRequestMethod("GET");
+
+    ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
+
+    byte[] buffer = new byte[1024];
+    int bytesRead;
+    InputStream inputStream = con.getInputStream();
+    while ((bytesRead = inputStream.read(buffer)) != -1) {
+      byteArrayOutputStream.write(buffer, 0, bytesRead);
```

Comment:
This could cause issue in case the input stream is consuming a part of an UTF-8 sequence (buffer size frontier reached)
UTF-8 sequence are encoded onto multi-bytes, so:
- glyph: `😎`
- UTF8 representation:  `F0 9F 98 8E` 

If reading only start of sequence `F0 9F` due to `buffer` 1024 size frontier: an unicode converter could throw an error or replace it with a invalid character.
So in this context; better to read the whole sequence then proceed to conversion